ui/src/cells/actions/thunks.ts

```diff
@@ -0,0 +1,162 @@
+// Libraries
+import {normalize} from 'normalizr'
+
+// APIs
+import * as api from 'src/client'
```

Comment:
i hate this. can we be more explicit for the same reasons we're using typescript? (to document surface area of an interface, but this time its a module)